build/configs/common_download.sh

```diff
@@ -296,17 +303,22 @@ download_all()
 		if [[ "${parts[$partidx]}" == "config" ]];then
 			continue
 		fi
-
+		if [[ "${parts[$partidx]}" == "resource" ]];then
+			if [[ $found_resource == true ]];then
+				continue
+			fi
+			found_resource=true
+		fi
 		exe_name=$(get_executable_name ${parts[$partidx]})
 		if [[ "No Binary Match" = "${exe_name}" ]];then
 			continue
 		fi
-
-		echo ""
-		echo "=========================="
-		echo "Downloading ${parts[$partidx]} binary"
-		echo "=========================="
-
+		if [[ $TTYDEV == *"USB"* ]]; then
```

Comment:
Is this to reduce download logs?

Reply:
USB checksum             ......    OK
Image download           ......    OK
common_rtl8730e_200204.trpk download to 0x824d000
USB erase                ......    OK
USB download             ......    OK
USB checksum             ......    OK
Image download           ......    OK
app1_rtl8730e_240510.trpk download to 0x8794000
USB erase                ......    OK
USB download             ......    OK
USB checksum             ......    OK
Image download           ......    OK
bootparam.bin download to 0x8ffe000
USB erase                ......    OK
USB download             ......    OK
USB checksum             ......    OK
Image download           ......    OK
rtk_ext_flash_data.bin download to 0x9000000
USB erase                ......    OK
USB download             ......    OK
USB checksum             ......    OK
Image download           ......    OK
resource_rtl8730e_240421.trpk download to 0x9020000

It can have confusion.. 
It is difficult to find which step error ocurring./
let's keep to print `=====` when this every partition

Reply:
=================, this depends on upload_linux_tools, right?